servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLAutoConfiguration.kt

```diff
@@ -30,4 +30,5 @@ import org.springframework.context.annotation.Import
     PlaygroundRouteConfiguration::class,
     SdlRouteConfiguration::class
 )
-class GraphQLAutoConfiguration
+class
+GraphQLAutoConfiguration
```

Comment:
i believe this change was unintentional?

Reply:
yes, sorry. Just removed it